HT3.py

```diff
@@ -0,0 +1,59 @@
+initial_text = r"""homEwork:
+	tHis iz your homeWork, copy these Text to variable. 
+
+	You NEED TO normalize it fROM letter CASEs point oF View. also, create one MORE senTENCE witH LAST WoRDS of each existING SENtence and add it to the END OF this Paragraph.
+
+	it iZ misspeLLing here. fix“iZ” with correct “is”, but ONLY when it Iz a mistAKE. 
+
+	last iz TO calculate nuMber OF Whitespace characteRS in this Text. caREFULL, not only Spaces, but ALL whitespaces. I got 87.
+"""
+import re

```

Comment:
I suggest bringing all imports up to the top according to pep8 standard (ref: https://peps.python.org/pep-0008/#imports)